nvme.c

```diff
@@ -6672,6 +6694,15 @@ static int copy_cmd(int argc, char **argv, struct command *cmd, struct plugin *p
 	nats = argconfig_parse_comma_sep_array_u32(cfg.elbats, elbats, ARRAY_SIZE(elbats));
 
 	nr = max(nb, max(ns, max(nrts, max(natms, nats))));
+	if (cfg.format == 2 || cfg.format == 3) {
+		if (nr != nid) {
+			nvme_show_error("formats 2 and 3 require source namespace ids for each source range");
+			err = -EINVAL;
+		}
+	} else if (nid > 0) {
+		nvme_show_error("formats 0 and 1 do not support cross-namespace copy");
+		err = -EINVAL;
```

Comment:
Should both these error cases `return`?

Reply:
Yes! Embarrassing that I didn't catch these on the rebase.